pkg/api/types.go

```diff
@@ -1518,6 +1518,9 @@ type PodSecurityContext struct {
 	//
 	// If unset, the Kubelet will not modify the ownership and permissions of any volume.
 	FSGroup *int64 `json:"fsGroup,omitempty"`
+	// The seccomp profile that containers in the pod will run with.  Any container
+	// security context not defining its own profile will inherit the pod level profile.
+	SeccompProfile *string `json:"seccompProfile,omitempty"`
```

Comment:
Looks like to be convention in all other api objects that `*string` is never used, but always `string`.

There is a difference for lists usually: empty list is different from nil-list. But for strings - at least in go - empty string is the unset string.